page.md

```diff
@@ -21,8 +21,7 @@ The PAGE XML root element `<pc:PcGts>` MUST have exactly one `<pc:Page>`.
 ### URL for imageFilename / filename
 
 The `imageFilename` of the `<pg:Page>` and `filename` of the
-`<pg:AlternativeImage>` element MUST be a URL. A local filename should be a
-`file://` URL.
+`<pg:AlternativeImage>` element MUST be a filename relative to the `mets.xml`.
 
 All URL used in `imageFilename` and `filename` [MUST be referenced in a fileGrp
 in METS](https://ocr-d.github.io/mets#if-in-page-then-in-mets).
```

Comment:
Use `https://ocr-d.de/mets` instead of `https://ocr-d.github.io/mets`

Reply:
I think I'll do those in a separate PR, there are a few more places.